src/Components/OATPropertyEditor/FormUpdateProperty.tsx

```diff
@@ -308,11 +309,13 @@ export const FormUpdateProperty = ({
     }, [multiLanguageSelectionsDescription]);
 
     const getIdErrorMessage = () => {
-        return idLengthError
+        const idWarn = idWarning ? t('OATPropertyEditor.warningId') : '';
+        const idError = idLengthError
             ? t('OATPropertyEditor.errorIdLength')
             : idValidDTMIError
             ? t('OATPropertyEditor.errorIdValidDTMI')
             : '';
+        return `${idWarn} ${idError}`;
```

Comment:
I'm not sure if concatting these messages together will make it particularly readable. We might find separate lines are better?

Reply:
We are using the Fluent UI textfield to display this https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield was looking at the doc and google a bit to see if I can make this multiline but didn't find a suitable answer, @jamescarpinter can you take a quick look I really can use your experience here

Reply:
Let me Talk to Jamie about this and will circle back to you

Reply:
@jamescarpinter Committed changes from ADTS-240 Into this PR let me know If anything else need to be done